pegasus_wrapper/pegasus_utils.py

```diff
@@ -21,7 +21,11 @@
 
 
 def path_to_pegasus_file(
-    path: Path, *, site: str = "local", name: Optional[str] = None
+    path: Path,
+    *,
+    site: str = "local",
+    name: Optional[str] = None,
+    is_raw_input: bool = False
```

Comment:
What does this parameter do?

Reply:
@LichMaster98

Reply:
This parameter indicates that the raw is an input which is always going to exist on the file system at the specified location. If so, we can safely add the PFP to the pegasus DAX.

Reply:
What distinguishes a raw input from a non-raw input?

Reply:
A raw input is a file that doesn't come from the output of another job in the workflow.

Reply:
Thanks, @LichMaster98 . @joecummings , can you add this information to the docstring?